.github/workflows/raedar_tests CI/raedar.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: raedar tests
```

Comment:
Maybe change the name format to raedar-acceptance-tests or something.

Reply:
the name has been changed to the recommended